README.md

```diff
@@ -1,21 +1,42 @@
+## What is the Label Studio ML backend?
+
+The Label Studio ML backend is an SDK that lets you wrap you machine learning code and turn it into a web server.
+You can then connect that server to a Label Studio instance to perform 2 tasks:
+
+- Dynamically pre-annotate data based on model inference results
+- Retrain or fine-tune a model based on recently annotated data
+
+If you need to just load static pre-annotated data into Label Studio, running an ML backend  might be overkill for you. Instead, you can [import preannotated data](https://labelstud.io/guide/predictions.html).
+
+## How it works
+
+<diagram>
+
+1. Get your model code
+2. Wrap it with the Label Studio SDK
+3. Create a running server script
+4. Launch script
+5. Connect Label Studio to ML backend on the UI
+
+
 ## Quickstart
 
 Here is a quick example tutorial on how to run the ML backend with a simple text classifier:
 
-0. Clone repo
+0. Clone the repo
    ```bash
    git clone https://github.com/heartexlabs/label-studio-ml-backend  
    ```
    
-1. Setup environment
+1. Set up environment
    ```bash
    cd label-studio-ml-backend
    pip install -e .
    cd label_studio_ml/examples
    pip install -r requirements.txt
    ```
    
-2. Create new ML backend
+2. Create an ML backend
```

Comment:
maybe we need to make it more obvious that creating an ML backend == launching the script that they just updated to run it as a web server basically

Reply:
yeah, make sense, could you add it please?